doc/tutorials/04-lattice_boltzmann/04-lattice_boltzmann_part1.ipynb

```diff
@@ -9,11 +9,12 @@
     "#### Before you start:\n",
```

Comment:
<p>I guess the equation should read </p><pre class="ql-syntax" spellcheck="false">\begin{align}
&nbsp;\rho &amp;= \sum_i f_i \
&nbsp;\vec{j} = \rho \\
&nbsp;\vec{u} &amp;= \sum_i f_i \vec{c_i} \\
&nbsp;\Pi^{\alpha \beta} &amp;= \sum_i f_i \vec{c_i}^{\alpha}\vec{c_i}^{\beta}
&nbsp;\label{eq:fields}
\end{align}
</pre><p>for better readability?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/3893/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>ReviewNB's support for LaTeX seems incomplete, the original equation looks good in Jupyter, with one equation per line</p>